# 5.
# Почему возникла ошибка? Объясните и исправьте, чтобы функция могла
# работать с данными на входе в примере.

def mult2(args):
    res = 1
    for i in args:
        res += i
    return res

List = [1, 2, 3, 4, 5, 6, 7]
print(mult2(List))